utt/util.py

```diff
@@ -44,8 +48,41 @@ def entries_from_file(filename):
         pass
 
 
+def localize(dt):
+    """ Given a naive datetime, localize it with the machine local timezone.
+
+    Parameters
+    ----------
+    dt : datetime.datetime
+        Naive datetime.
+
+    Returns
+    -------
+    datetime : datetime.datetime
+        Timezone-aware datetime for the local timezone.
+        If failed to obtain a local timezone, UTC is assumed.
+
+    Raises
+    ------
+    pytz.InvalidTimeError
+        If the given datetime is ambiguous or does not exist for the local
+        time zone,
+    """
+    try:
+        tz = tzlocal.get_localzone()
+    except pytz.UnknownTimeZoneError as exc:
+        warnings.warn("Unable to obtain local timezone. Assume UTC. "
+                      "Error: {}".format(str(exc)))
+        return pytz.UTC.localize(dt)
```

Comment:
In case when local zone cannot be determined, istead of assuming UTC and emitting a warning, we could raise here with a message telling the user what they can do to fix it: e.g. allow setting an environment variable with a timezone name recogized by `pytz`